keccak256/src/keccak_arith.rs

```diff
@@ -106,6 +106,7 @@ impl KeccakFArith {
     ) -> StateBigInt {
         if let Some(next_input) = next_input {
             let out_1 = KeccakFArith::absorb(a, next_input);
+            // TODO: Base conversion missing
```

Comment:
Will this be addressed in another PR?

Reply:
That's a leftover.. Sorry.
Removed and merging!